### DISCUSSION TITLE
Installation - Manage Node

### DISCUSSION THREAD

**Discussion by NetworkGuyMark** [Security-Onion-Solutions/securityonion#2562] [Unsupported Versions]
I've been through the SO Essentials Training and have started going through the paid training.

I've downloaded the newest ISO, built a VM in ESXi with the following specs:

Distributed - Manage Node

6 CPU, 20 GB RAM, 400 GB HD and 1 NIC.

I go through the entire installation/setup process with no issues.  I choose STATIC IP and give it my IP, Subnet, etc.  On reboot the IP I assigned via STATIC during setup was not used... instead there is a DHCP assigned.  I assume something didn't work during setup.  I am sure its easy enough to change to a static for the network setup, BUT... is this an issue with any other components from the install?  Any idea what would be the problem as to why setup didn't use static?  Could there be other issues with the install?

Any help would be greatly appreciated.

Mark

**Comment by TOoSmOotH** [comment]
Does the /root/sosetup.log have any clues? I am unable to reproduce this.
